contracts/test/MockOracle.sol

```diff
@@ -1,10 +1,10 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.6.6;
+pragma solidity >=0.8.0;
 
-import "@chainlink/contracts/src/v0.6/LinkTokenReceiver.sol";
-import "@chainlink/contracts/src/v0.6/interfaces/ChainlinkRequestInterface.sol";
-import "@chainlink/contracts/src/v0.6/interfaces/LinkTokenInterface.sol";
-import "@chainlink/contracts/src/v0.6/vendor/SafeMathChainlink.sol";
+import "@chainlink/contracts/src/v0.8/operatorforwarder/dev/LinkTokenReceiver.sol";
+import "@chainlink/contracts/src/v0.8/interfaces/ChainlinkRequestInterface.sol";
+import "@chainlink/contracts/src/v0.8/shared/interfaces/LinkTokenInterface.sol";
+import "./SafeMathChainlink.sol";
```

Comment:
This used to be vendored, but wondering whether we need it anymore.  I tried replacing line 84: uint256 expiration = (block.timestamp).add(EXPIRY_TIME);
uint256 expiration = (block.timestamp + EXPIRY_TIME);